lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -6632,6 +6622,140 @@ LogicalResult ConvertAtenOp<AtenDiagEmbedOp>::matchAndRewrite(
   return success();
 }
 
+// Legalization for aten.uniform
+// Since TOSA hasn't got a built-in random generator yet, we will use
+// std::uniform_real_distribution with the std::default_random_engine from C++
+// <random> library
+template <>
+LogicalResult ConvertAtenOp<AtenUniformOp>::matchAndRewrite(
```

Comment:
Does this need a LIT test too or since it's covered by e2e tests the LIT one can be skipped?

Reply:
Thank you for the comment, I will add a LIT test for that op too.